packages/manager/modules/pci/src/projects/project/instances/instances.service.js

```diff
@@ -567,33 +569,39 @@ export default class PciProjectInstanceService {
     return null;
   }
 
-  getExtraBandwidthCost(project, user) {
-    return this.OvhApiOrderCatalogPublic.v6()
-      .get({
-        productName: 'cloud',
-        ovhSubsidiary: user.ovhSubsidiary,
+  getCatalog(endpoint, user) {
```

Comment:
Duplicate: As commented earlier. This method is defined in other places. Can you reuse it?

Reply:
Probably it's the case, currently i can't.
But each time i work on something i do some refactoring.

Probably for next times.